generators/generator-base.js

```diff
@@ -262,29 +270,41 @@ Generator.prototype.getAllInstalledLanguages = function () {
  * get all the languages supported by JHipster
  */
```

Comment:
I think that would be better. I guess signature changed only for `addMessageformatLocaleToIndex` method